wallet/wallet.c

```diff
@@ -2883,7 +2902,10 @@ u32 wallet_transaction_height(struct wallet *w, const struct bitcoin_txid *txid)
 		return 0;
 	}
 
-	blockheight = db_column_int(stmt, 0);
+	if (!db_column_is_null(stmt, 0))
+		blockheight = db_column_int(stmt, 0);
+	else
+		blockheight = 0;
```

Comment:
Maybe a db_column_int_or_null() helper for this pattern?

Reply:
I had a branch with `db_column_type_or_default()` at some point which I may dig up again if this happens alot.